src/I18n/RelativeTimeFormatter.php

```diff
@@ -29,23 +30,26 @@ class RelativeTimeFormatter implements DifferenceFormatterInterface
     /**
      * Get the difference in a human readable format.
      *
-     * @param \Cake\Chronos\ChronosInterface $dateTime1 The datetime to start with.
-     * @param \Cake\Chronos\ChronosInterface|null $dateTime2 The datetime to compare against.
+     * @param \Cake\Chronos\Chronos|\Cake\Chronos\ChronosDate $first The datetime to start with.
+     * @param \Cake\Chronos\Chronos|\Cake\Chronos\ChronosDate|null $second The datetime to compare against.
      * @param bool $absolute Removes time difference modifiers ago, after, etc.
      * @return string The difference between the two days in a human readable format.
-     * @see \Cake\Chronos\ChronosInterface::diffForHumans
+     * @see \Cake\Chronos\Chronos::diffForHumans
      */
     public function diffForHumans(
-        ChronosInterface $dateTime1,
-        ?ChronosInterface $dateTime2 = null,
+        Chronos|ChronosDate $first,
+        Chronos|ChronosDate|null $second = null,
```

Comment:
```suggestion
        Chronos|ChronosDate|DateTimeInterface|null $second = null,
```

We should allow 2nd argument to be native instance too since `Chronos:diff()` supports it. The types should be updated for the interface too.